src/memory/AmdFlash.cc

```diff
@@ -214,6 +215,11 @@ void AmdFlash::softReset()
 {
 	cmdIdx = 0;
 	setState(State::IDLE);
+	clearStatus();
+}
+
+void AmdFlash::clearStatus()
+{
 	status &= 0xC4;
```

Comment:
In general I agree with small helper functions. Even if only called once.
But maybe this is a bit extreme ;-)     (a comment `// clear status` could also work)
But if you like it, that's fine for me.

Reply:
Consider it gone.